src/modules/CRUD/components/Details/Read/AttachmentView/Attachment/Attachment.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Attachment = ({ className, file, legend }) => {
+  if (!file) {
+    return null;
+  }
+  return (
+    <div className={className}>
+      <a rel="noopener noreferrer download" target="_blank" href={file}>{legend}</a>
+    </div>
+  );
+};
```

Comment:
```suggestion
const Attachment = ({ file, legend, ...props }) => {
  if (!file) {
    return null;
  }
  return (
    <div {...props}>
      <a rel="noopener noreferrer download" target="_blank" href={file}>{legend}</a>
    </div>
  );
};
```

Reply:
At the beginning, i also forwarded props on div, but 
1/ I don't need other props on this div, but `className`
2/ It's set a lot of unecessary props on this node like the following example  
`<div class="attachment-item__file" id="4" category="1" action_url="/api/crud/features/b40312d8-6391-4659-9d84-e64a088c2ba2/attachments/4/" created_at="16-04-2020 12:21" updated_at="17-04-2020 17:33">`  
(vs `<div class="attachment-item__file">`)

Reply:
I think one shouldn't pass to a component, props that ain't used by the component nor any child component.

Discussion subject seems to be the same as https://github.com/Terralego/terra-admin/pull/238#discussion_r411347577

Reply:
True, but these props come from an API response i store. I am not sure it's efficient to filter these each time i used it. 

Moreover, in the future i should use some of these (action_url, id, and category) to CRUDify each item.

Reply:
I agree with the idea to try to filter what we give to a component, I'm not sure this is a big overhead to filters the props each time we get data from API (it's just a de-structuration and we do it all the time) but it definitely make the component more readable and the component use also. You immediately know that the Attachment component use only file and legend props.

It's hard for a reviewer to knows what props is really used when you give a component a bunch of data coming from API. I read too many code like that in others Terralego projects and it's not fun, believe me ;-)

However, this component is interchangeable with another one (L.13 of AttachmenLlist.js), so it may seem more delicate to filter all the properties you don't want if the other component uses them. The example here is "thumbnail" which is only useful for the "Picture" component.